modules/python/Makefile.am

```diff
@@ -69,6 +70,28 @@ modules_python_libmod_python_la_DEPENDENCIES = $(MODULE_DEPS_LIBS)
 
 modules/python mod-python: modules/python/libmod-python.la
 
+else
+
+module_LTLIBRARIES += modules/python/libmod-python2.la
+modules_python_libmod_python2_la_CFLAGS = $(python_common_cflags) $(PYTHON2_CFLAGS)
+modules_python_libmod_python2_la_CPPFLAGS = -DSYSLOG_NG_ENABLE_PYTHONv2=1 $(python_common_cppflags)
```

Comment:
They can't. When `--enable-python2-and-3` is used, nor `SYSLOG_NG_ENABLE_PYTHONv2` neither `SYSLOG_NG_ENABLE_PYTHONv3` is set, it has to be set here in our Makefile.

That was my idea to avoid additional complex ifdef branching in our codebase.